src/main/java/com/example/petbuddybackend/service/mapper/ChatMapper.java

```diff
@@ -14,4 +20,15 @@ public interface ChatMapper {
     @Mapping(target = "senderEmail", source = "sender.email")
     @Mapping(target = "chatId", source = "chatRoom.id")
     ChatMessageDTO mapToChatMessageDTO(ChatMessage chatMessage);
+
+    @Mapping(target = "createdAt", source = "createdAt", qualifiedByName = "mapToZonedDateTime")
+    ChatMessageDTO mapTimeZone(ChatMessageDTO chatMessage, @Context ZoneId zoneId);
```

Comment:
Ja bym to przerzucil po prostu do mappera gornego "mapToChatMessageDTO"  i dodal ZoneId zeby zmapowalo razem. Jak uzytkownik nie poda strefy to i tak zmapuje na default, a zrobimy to wywolaniem 1 metody zamiast 2.

Reply:
Coś się ogarnie, na pewno da się poupraszczać te zoneId